config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'user#list'
+  get 'user/show'
+  get 'post/list'
+  get 'posts/show'
```

Comment:
- [ ] We are required to only define the `index` and `show` actions for the User and Post resources. Additionally, the post routes should be nested within the user routes. Kindly refactor your routes

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```